SU-HSDO/suhumsci
# SHS-4652: Bug: hb-raised-cards utility classes are affecting Vertical Linked Cad styling

# READY FOR REVIEW

## Summary
- Fixes conflicts with `.hb-raised-cards` utility class with `hb-raised-cards--uniform-height` and `hb-stretch-vertical-linked-cards`

## Need Review By (Date)
3/20

## Urgency
low

## Steps to Test
1. Spin up both Colorful and Traditional for testing. The results for each will be slightly different.
2. Go to https://hs-colorful.suhumsci.loc/collections-cards and https://hs-traditional.suhumsci.loc/collections-cards
3. Click the "Layout" tab and "Configure" the main region that houses the cards. Add `hb-raised-cards hb-stretch-vertical-linked-cards hb-raised-cards--uniform-height` as your classes. Note both the "stretch" and "uniform" clases are added here. The results should look like: 
![Screenshot from 2023-03-17 09-50-38](https://user-images.githubusercontent.com/369018/225939688-625c9760-32ad-481a-85d8-9522c982c398.png)
![Screenshot from 2023-03-17 09-51-03](https://user-images.githubusercontent.com/369018/225939709-d10dc264-ea54-4049-b244-3f11f2898385.png)

4. Edit the layout classes again but remove the "uniform" class. Results should be:
![Screenshot from 2023-03-17 09-54-26](https://user-images.githubusercontent.com/369018/225940491-35901a0e-8648-43d3-bcc1-60452c71a7be.png)
![Screenshot from 2023-03-17 09-54-15](https://user-images.githubusercontent.com/369018/225940508-ef9dc92e-234e-4336-95f3-c6999e6f4e39.png)

5. Edit the layout classes again but restore the "stretch" class and remove the "uniform" class. The results should be:
![Screenshot from 2023-03-17 09-56-47](https://user-images.githubusercontent.com/369018/225941013-71cc8727-c734-4281-b204-31c43fba7140.png)
![Screenshot from 2023-03-17 09-57-27](https://user-images.githubusercontent.com/369018/225941413-2646b2d8-dfb0-4cbb-ae23-c664a74d99ec.png)



## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
